frontend/src/utils/utils.ts

```diff
@@ -63,6 +64,27 @@ export function partition<T>(
   );
 }
 
+/**
+ * @function excludeMetaTag
+ * Filter out a configured list of select metadata or tags from a COMS response
+ * @param {object} type either 'metadata' or 'tagset'
+ * @param {object} axiosResponse A resolved axios response from COMS
+ * @returns {object} The response data with select metadata/tags from a configured list removed
+ */
+export function excludeMetaTag(type: string, data: any){
```

Comment:
- As excludeMetaTag is a new function, don't forget to add unit tests for this utility.
- Wondering if we should be considering using an enum type for this instead of just a string, as the function name implies only metadata or tags, whereas we accept basically any valid string here. This can easily lead to an error (that we do catch with the ?? operator) if we pass in a string that doesn't exist in the exclude object.